azure-pipelines.yml

```diff
@@ -101,6 +101,9 @@ stages:
           projectName: '$(sonarCloudMsBuildProjectName)'
           projectVersion: '$(SONAR_PROJECT_VERSION)'
           scannerMode: MSBuild
+          extraProperties: |
+            sonar.cs.opencover.reportsPaths="$(Agent.TempDirectory)/coverage/**.xml"
```

Comment:
did you change to opencover?

Reply:
https://community.sonarsource.com/t/coverage-test-data-generate-reports-for-c-vb-net/9871

 sonar.cs.vscoveragexml.reportsPaths for Visual Studio Code Coverage